binstar_client/inspect_package/pypi.py

```diff
@@ -229,8 +230,10 @@ def inspect_pypi_package_sdist(filename, fileobj):
         distrubite = True
         if data is None:
             raise errors.NoMetadataError("Could not find *.egg-info/PKG-INFO file in pypi sdist")
-
-    config_items = Parser().parsestr(data).items()
+    if sys.version_info.major==3:
```

Comment:
One optional suggestion I have is to refactor the parse functionality into a separate function rather than duplicate the same fix twice.